### PR TITLE
Make Family Ubudehe Question Optional

### PR DESCRIPTION
**Pull request by Diama1**
#1307

### CODE REVIEW

**Comment by Diama1** [comment]
[ubudehe.webm](https://github.com/user-attachments/assets/f87f5b4c-bf6e-4e5b-a122-61c30187460d)

File: client/src/elm/Activity/Utils.elm [hunk 1]
```diff
@@ -467,18 +467,7 @@ expectMotherActivity currentDate offlineSession motherId activity =
...
-                                                            Just Ubudehe1 ->
-                                                                True
-
-                                                            Just Ubudehe2 ->
-                                                                True
-
-                                                            _ ->
-                                                                False
-                                                    )
+                                                |> Maybe.map (\_ -> True)
```
Comment (anvmn) [inline client/src/elm/Activity/Utils.elm]: If there's no more dependency on Ubudehe, and every mother should get it, drop the condition.

**Review by anvmn — CHANGES_REQUESTED** [review]
@Diama1 drop the condition

File: client/src/elm/Activity/Utils.elm (hunk 1 above)
Comment (Diama1) [reply client/src/elm/Activity/Utils.elm]: @anvmn there is no dependency on Ubudehe

Comment (anvmn) [reply client/src/elm/Activity/Utils.elm]: @Diama1 then drop the condition

File: client/src/elm/Activity/Utils.elm [hunk 2]
```diff
@@ -464,24 +464,8 @@ expectMotherActivity currentDate offlineSession motherId activity =
...
-                                                            Just Ubudehe2 ->
-                                                                True
-
-                                                            _ ->
-                                                                False
-                                                    )
-                                                |> Maybe.withDefault False
                                     in
-                                    isBreastfeeding && entitledByUbudehe
+                                    isBreastfeeding
```
Comment (anvmn) [inline client/src/elm/Activity/Utils.elm]: @Diama1 now we don't need to have it as a param.
Remove the `let`, and apply logic directly.

**Review by anvmn — APPROVED** [review]

**Pull request merged**